includes/classes/PullListTable.php

```diff
@@ -247,6 +250,17 @@ public function column_date( $post ) {
 		}
 	}
 
+	/**
+	 * Output categories column
```

Comment:
a small suggestion about the full stop.

```suggestion
	 * Output categories column.
```

Reply:
Done